app.js

```diff
@@ -16,13 +20,15 @@ function onOpen (soc, channelIDs) {
     msgBroker.init();
 
     channelIDs.forEach(function(id) {
-        msgBroker.push({
-            "id": 1,
-            "type": "message",
-            "channel": id,
-            "text": "WhoopBot " + process.env.VERSION +
-                " connected to WebSocket"
-        });
+        if (!_.includes(whitelistChannels, id)) {
+            msgBroker.push({
+                "id": 1,
+                "type": "message",
+                "channel": id,
+                "text": "WhoopBot " + process.env.VERSION +
+                    " connected to WebSocket"
+            });
+        }
```

Comment:
Yes, did I invert what whitelisted means? I was thinking that whitelistedChannels would be a list of banned channels; do you think it should be a list of allowed channels?